paddle/framework/.clang-format

```diff
@@ -0,0 +1,5 @@
+---
+Language:        Cpp
```

Comment:
Not sure, just remind here.

It will scan [static analysis] and trigger the error if your code is "out of the box", for instance, if you did not add `// namespace platform` comment at the end, here is a buggy.